CMakeLists.txt

```diff
@@ -557,8 +566,9 @@ find_package(Threads REQUIRED)
 set(MFEM_TPLS OPENMP HYPRE LAPACK BLAS SuperLUDist STRUMPACK METIS SuiteSparse
     SUNDIALS PETSC SLEPC MUMPS AXOM FMS CONDUIT Ginkgo GNUTLS GSLIB
     NETCDF MPFR PUMI HIOP POSIXCLOCKS MFEMBacktrace ZLIB OCCA CEED RAJA UMPIRE
-    ADIOS2 CUSPARSE MKL_CPARDISO MKL_PARDISO AMGX CALIPER CODIPACK
-    BENCHMARK PARELAG TRIBOL MPI_CXX HIP HIPSPARSE MOONOLITH BLITZ ALGOIM ENZYME)
+    ADIOS2 CUSPARSE MKL_CPARDISO MKL_PARDISO AMGX MAGMA CALIPER CODIPACK
+    BENCHMARK PARELAG TRIBOL MPI_CXX HIP HIPBLAS HIPSPARSE MOONOLITH BLITZ
+    ALGOIM ENZYME)
```

Comment:
Add `CUBLAS` here and move them with `CUSPARSE` after `MAGMA` -- this way we may actually be fine to remove the CUDA/HIP logic from `FindMAGMA.cmake` (unless we want to have a check that the MAGMA build uses the same GPU library as MFEM, CUDA or HIP).

Reply:
Done in bb8a5b88d.